README.md

```diff
@@ -1 +1,73 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+This is a first project from microverse
+
+## Built With
+
+- npm
+- vscode
+- html
+- css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] I would like to recommend you remove this part from your readme. You readme need to showcase only necessary information about your project. PLease remove it in order to make your readme more professional  :+1: